packages/cli/src/commands/destroy.ts

```diff
@@ -8,18 +9,34 @@ export default class Destroy extends AuthCommand {
   static hidden = false
   static description = 'Destroy your project'
 
+  static flags = {
+    force: Flags.boolean({
+      char: 'f',
+      description: 'force mode',
+      default: false,
+    }),
+    config: Flags.string({
+      char: 'c',
+      description: 'The Checkly CLI config filename.',
+      allowNo: true,
+    }),
+  }
+
   async run (): Promise<void> {
-    const cwd = process.cwd()
-    const { config: checklyConfig } = await loadChecklyConfig(cwd)
+    const { flags } = await this.parse(Destroy)
+    const { force, config: configFilename } = flags
+    const { config: checklyConfig } = await loadChecklyConfig(configFilename)
     const { data: account } = await api.accounts.get(config.getAccountId())
-    const { projectName } = await prompt([{
-      name: 'projectName',
-      type: 'test',
-      message: `Are you sure you want to delete all resources in project "${checklyConfig.projectName}" for account "${account.name}"?\n  Please confirm by typing the project name "${checklyConfig.projectName}":`,
-    }])
-    if (projectName !== checklyConfig.projectName) {
-      this.log(`The entered project name "${projectName}" doesn't match the expected project name "${checklyConfig.projectName}".`)
-      return
+    if (!force) {
+      const { projectName } = await prompt([{
+        name: 'projectName',
+        type: 'test',
```

Comment:
This really didn't have to be in this PR

Reply:
Let's push this separately. The main complexity of this PR is the config flag and the project directory

Reply:
@umutuzgur , I added here because allows me to execute the `destroy` commands E2E tests. I could move all the `destroy` command tests with the `--force` flag to a new PR if you prefer.

Reply:
@umutuzgur , I removed the `--force` flag for the `destroy` command. I'll add it (including the destroy E2E tests) after merging these changes.